src/ui/customBreakpointsUI.ts

```diff
@@ -93,34 +124,94 @@ export function registerCustomBreakpointsUI(
 ) {
   const provider = new BreakpointsDataProvider(debugSessionTracker);
 
-  vscode.window.createTreeView(CustomViews.BrowserBreakpoints, {
+  const view = vscode.window.createTreeView(CustomViews.EventListenerBreakpoints, {
     treeDataProvider: provider,
+    showCollapseAll: true,
   });
+
+  view.onDidChangeCheckboxState(e => {
+    if (e.items.length == 1 && e.items[0][0] instanceof vscode.TreeItem) {
+      if (!e.items[0][0].label) return;
+      const allBr = provider.breakpoints.get(e.items[0][0].label.toString());
+      if (!allBr) return;
+      if (e.items[0][1]) {
+        provider.addBreakpoints(allBr);
+      } else {
+        provider.removeBreakpoints(allBr);
+      }
+      return;
+    }
```

Comment:
I'm still not a fan of special-casing the "if there's exactly one change in this event". VS Code doesn't guarantee that group selections or deselections are presented this way. Instead we could check `if (item instanceof Breakpoint)` in add/removeBreakpoints and do the necessary logic on children there